package.json

```diff
@@ -9,6 +9,7 @@
     "dotenv": "^8.2.0",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.15.1",
+    "graphql-markdown": "^5.2.0",
```

Comment:
I don't think we need this, this was probably you testing, right? Let's remove it please.